Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM rust:1.67 AS build
+ARG CARGO_FLAGS="--all-features --release"
+
+WORKDIR /app
+COPY . ./
+RUN cargo build ${CARGO_FLAGS}
+
+FROM gcr.io/distroless/cc
+COPY --from=build /app/target/release/stylua /
+CMD ["./stylua"]
```

Comment:
I don't know too much about Docker, but is it worth configuring an `ENTRYPOINT` as well? It seems that allows it to be used through `docker run`

Reply:
You can still use `docker run` with `CMD`, I think this is the right way. From what I understand `ENTRYPOINT` is more for when you still want the `ENTRYPOINT` to run in addition to specifying an additional command which I don't think is the case for a CLI tool.

You can try like this:

```
$ docker build . -t stylua
$ docker run stylua /stylua "*.lua"
```